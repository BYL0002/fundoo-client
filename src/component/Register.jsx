import React from 'react';
import TextFieldComponent from '../component/TextfieldComponent';
import { Button } from '@material-ui/core';

class Register extends React.Component {
    render() {
        return (
            <div className = "Form">
                <TextFieldComponent name = "email" />
                <Button>Submit</Button>
            </div>
        )        
    }
}

export default Register;