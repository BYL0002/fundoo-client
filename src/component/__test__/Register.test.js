import React from 'react';
import { shallow } from 'enzyme';
import Register from '../Register';

// import { configure } from 'enzyme';
// import Adapter from 'enzyme-adapter-react-16';
// configure({ adapter: new Adapter() });

describe('Register', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<Register debug />);
  
    expect(component).toMatchSnapshot();
  });
});