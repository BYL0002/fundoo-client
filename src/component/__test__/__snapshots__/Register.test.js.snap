// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Register should render correctly in "debug" mode 1`] = `
<div
  className="Form"
>
  <div
    className="formHeader"
  >
    Register
    <span
      className="beforeDashboardTitle"
    >
      Fundoo Notes
    </span>
  </div>
  <div>
    <div>
      <TextField
        className="nameTextFieldRegisterPage"
        label="Name"
        name="name"
        onChange={[Function]}
        required={false}
        select={false}
        value=""
        variant="standard"
      />
    </div>
    <div>
      <TextField
        className="emailTextFieldRegisterPage"
        label="Email"
        name="email"
        onChange={[Function]}
        required={false}
        select={false}
        value=""
        variant="standard"
      />
    </div>
    <div>
      <WithStyles(Button)
        className="registerButtonRegisterPage"
        color="primary"
        onClick={[Function]}
        variant="extendedFab"
      >
        Submit
      </WithStyles(Button)>
    </div>
    <div>
      <span
        className="CenterTextStyle"
      >
        Have an account? 
      </span>
      <a
        className="links"
        href="/"
      >
         
        <b>
          Login
        </b>
         
      </a>
    </div>
  </div>
</div>
`;
